### PR TITLE
Spring Web: support for setting the ActorSystem name

### PR DESCRIPTION
**Pull request by sleipnir**
This pull request is intended to allow the ActorSystem name to be configurable.
The discussion related to this PR can be found at https://discuss.lightbend.com/t/alpakka-spring-web-support-for-setting-the-actorsystem-name/3273

### CODE REVIEW

**Comment by sleipnir** [comment]
It seems to me that the build process broke down into an unrelated task.
Someone with some idea?

File: spring-web/src/test/java/docs/javadsl/SampleController.java [hunk 1]
```diff
@@ -5,20 +5,39 @@
...
+
   @RequestMapping("/")
   public Source<String, NotUsed> index() {
     return Source.repeat("Hello world!").intersperse("\n").take(10);
   }
+
+  @PostConstruct
+  public void setup() {
+    LoggingAdapter log = system.log();
+	  log.info("Injected ActorSystem Name -> {}", system.name());
```
Comment (2m) [inline spring-web/src/test/java/docs/javadsl/SampleController.java]: Did a TAB character sneak in here? :)

**Comment by 2m** [comment]
Unrelated test failure was https://github.com/akka/alpakka/issues/1360

**Comment by sleipnir** [comment]
@2m That's what I thought :)

**Pull request merged**